Removed redundant text in doc (#248)
freechipsproject/chisel-testers: README.md

circuit, or device under test (DUT) from value vectors for each input.  Testing the outputs each cycle against a separate
set of value vectors for each output.

> For a longer descripton see the [Using the PeekPokeTester](https://github.com/ucb-bar/chisel-testers/wiki/Using%20the%20PeekPokeTester)

Reviewer: ```suggestion
For a longer description, see the [Using the PeekPokeTester](https://github.com/ucb-bar/chisel-testers/wiki/Using%20the%20PeekPokeTester).
```

> For a longer descripton see the [Using the PeekPokeTester](https://github.com/ucb-bar/chisel-testers/wiki/Using%20the%20PeekPokeTester)

Reviewer: ```suggestion
For a longer description, see the [Using the PeekPokeTester](https://github.com/ucb-bar/chisel-testers/wiki/Using%20the%20PeekPokeTester).
```

> For a longer description, see the [Using the PeekPokeTester](https://github.com/ucb-bar/chisel-testers/wiki/Using%20the%20PeekPokeTester).

Reviewer: ```suggestion
For a longer description, see the [Using the PeekPokeTester](https://github.com/freechipsproject/chisel-testers/wiki/Using%20the%20PeekPokeTester).
```

Author: Yep totally agree^_^
